rules/ensure-automount-service-account-is-false/raw.rego

```diff
@@ -0,0 +1,21 @@
+package armo_builtins
+
+# deny if default ServiceAccount automountServiceAccountToken is set to true
+deny [msga]{
+
+    sa := input[_]
+	sa.kind == "ServiceAccount"
+	sa.metadata.name == "default"
+	sa.automountServiceAccountToken == true
+
+    msga := {
+	    "alertMessage": "Default ServiceAccount automountServiceAccountToken is set to True",
+		"alertScore": 9,
+		"packagename": "armo_builtins",
+		"failedPaths": ["automountServiceAccountToken: true"],
+		"fixPaths": [{"path": "automountServiceAccountToken",  "value": "false"}],
```

Comment:
You need to return either failpath or fixpath, according to the specific case.
This is true for this rule since you are checking one configuration which either exists, or doesn't, in the yaml. We have other rules where the rule response may contain both failpaths and fixpaths.

Reply:
irelevant since rule was changed